src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs

```diff
@@ -94,252 +92,7 @@ protected override void CheckLabel(RoleMappedData data)
 
         protected override void ComputeTrainingStatistics(IChannel ch, FloatLabelCursor.Factory cursorFactory, Float loss, int numParams)
```

Comment:
>ComputeTrainingStatistics [](start = 32, length = 25)

Poisson regression just do nothing, maybe it's better to just do nothing than throw exception.

Reply:
actually, MKL needed only for stdErrors, so maybe we should omit them and return other statistics?

And in general it would be shame to just delete code, someone probably put lot of effort in writing it, and it gonna get lost after you check-in

---
In reply to: [206357557](https://github.com/dotnet/machinelearning/pull/611#discussion_r206357557) [](ancestors = 206357557)

Reply:
I did create this to track introducing a Utility that can reference MKL and produce full stats:

[https://github.com/dotnet/machinelearning/issues/612](https://github.com/dotnet/machinelearning/issues/612)

But you are right, partial stats is better than no stats. Thanks for the suggestion. 

---
In reply to: [206358011](https://github.com/dotnet/machinelearning/pull/611#discussion_r206358011) [](ancestors = 206358011,206357557)